resources/js/src/app/components/itemList/filter/ItemRangeSlider.vue

```diff
@@ -179,9 +180,9 @@ export default defineComponent({
           }
         });
 
-        slider.on('update', function (ui: string[]) {
-          valueFrom.value = ui[0];
-          valueTo.value = ui[1];
+        slider.on('update', function (values: (number | string)[]) {
+          valueFrom.value = values[0];
+          valueTo.value = values[1];
```

Comment:
As this was only `string[]` before, do we need to parse the numbers to strings now?

Reply:
With the current config probably not, could add a check if needed?

Reply:
Let's just use `values[0].toString()`, the function exists for numbers and strings